gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/longrunning/stub/HttpJsonOperationsStub.java

```diff
@@ -72,8 +72,14 @@
 public class HttpJsonOperationsStub extends OperationsStub {
   private static final Pattern CLIENT_PACKAGE_VERSION_PATTERN =
       Pattern.compile("\\.(?<version>v\\d+[a-zA-Z]*\\d*[a-zA-Z]*\\d*)\\.[\\w.]*stub");
-
-  private static final ApiMethodDescriptor<ListOperationsRequest, ListOperationsResponse>
+  private static final String LRO_LIST_OPERATIONS = "google.longrunning.Operations.ListOperations";
```

Comment:
Are these all possible operations? Judging from the code in the composer, looks like there could be patch etc.?

Reply:
There are only 5 operations (and 4 of them have http annotations): https://github.com/googleapis/googleapis/blob/c8a382b9980b59a241a561aef1599980209a5e87/google/longrunning/operations.proto#L67-L122